src/components/WorkOrders/CloseWorkOrderByProxy.js

```diff
@@ -85,7 +84,7 @@ const CloseWorkOrderByProxy = ({ reference }) => {
       })
 
       setWorkOrder(new WorkOrder(workOrder))
-      setIsOvertime(workOrder.isOvertime)
+      workOrder.paymentType && setPaymentType(workOrder.paymentType)
```

Comment:
The presence of this key in the response needs to be verified with the backend team.